### ISSUE TITLE
Hyper .editorconfig

### ISSUE THREAD

**Issue by mathiasbno** [hyperoslo/playbook#68]
So we should have a Hyper `.editorconfig` template to use in our projects to make it easier to take over project and in general just streamline development, corporate here we go!

Some already have their own file in projects you are working on but we should maybe have ether a default in the playbook or a separate project you can copy and paste from in the start of the project. This is so we make sure to collect all additions for different languages and global settings to obeyed by.

What do you guys think?

Not familiar with `.editorconfig`, no problem, read all about it here: http://editorconfig.org/.

**Comment by timkurvers** [comment]
👍

**Comment by sindrenm** [comment]
I agree with this. But it's worth keeping in mind that different languages might have different standards. For instance, in Java, four spaces is pretty much the gold standard for indentation (and I think Kotlin pretty much adopted that, even though I personally prefer two spaces). So having one editor config file saying that we should always do four spaces might not fly well cross-team. :smile:
